src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -290,6 +294,45 @@ protected function getFormActions()
                     ->setUseButtonTag(true)
                     ->addExtraClass('btn-outline-danger btn-hide-outline font-icon-trash-bin action--delete'));
             }
+
+            $gridStateStr = $this->getRequest()->requestVar('gridState');
```

Comment:
Could you use a different variable name, or expand `Str` to whatever it's longer form should be?